### DISCUSSION TITLE
Is it possible to change kit main color ?

### DISCUSSION THREAD

**Discussion by jacopo-degattis** [johnlindquist/kit#897] [Q&A] [answered]
Hi everyone,
I was wondering: is it possible to change color theme on kit ?
Let me know
Thanks in advance !

**Comment by johnlindquist** [comment, marked as answer]
Technically, yes. I'm still working on exposing a way to do it to users though. 

I'd be _really_ curious how you would expect to be able to change the theme...

I think I invited you to the discord server before, but it would be a great place to chat about how we can build this out: https://discord.gg/tAnTJ6j8

**Reply by jacopo-degattis** [reply]
That's perfect, meet you on discord then !

**Reply by Ambushfall** [reply]
I have a workaround, but I'd still love to be added to the discord if possible :D

**Reply by johnlindquist** [reply]
@Ambushfall Here you go:

https://discord.gg/3nxaTHM2
